endpoints/contractors.md

```diff
@@ -0,0 +1,133 @@
+# Contractors
+
+- [Create contractors](#create-contractors)
+
+## Create contractors
+
+`POST /public/v1/contractors` creates a new contractor with the provided information.
+
+Please note:
+
+- Content-Type header `Content-Type: application/json` is required when POSTing contractors into Gearbox.
+- Snake case object keys are required.
+
+### Request
+
+```
+URL: https://api.gearbox.com.au/public/v1/contractors
+Method: POST
+Content-Type: application/json
+Authorization: Bearer $ACCESS_TOKEN
+
+{
+  company_name: "", // string, required, maximum length 100 characters
+  terminated: "", // boolean, optional, defaults to false
+  phone: "", // string, optional, maximum length 20 characters
+  mobile: "", // string, optional, maximum length 20 characters
+  address1: "", // string, optional, maximum length 100 characters
+  address2: "", // string, optional, maximum length 100 characters
+  city: "", // string, optional, maximum length 100 characters
+  state: "", // string, optional, maximum length 20 characters
+  postcode: "", // string, optional, maximum length 10 characters
+  abn: "", // string, optional, maximum length 100 characters
+  email: "", // string, optional, maximum length 50 characters
+  fax: "", // string, optional, maximum length 20 characters
+  contact: "", // string, optional, maximum length 50 characters
+  notes: "", // string, optional
+  code: "", // string, optional, maximum length 20 characters
+  position: "", // string, optional
+  vehicle_group: "" // string, optional, must match existing Vehicle Group
```

Comment:
Minor nitpick but can you fix the layout so the comments are inline and separated from the fields?

<img width="1013" alt="Screen Shot 2022-11-21 at 9 57 30 am" src="https://user-images.githubusercontent.com/16198924/202931150-6c2118db-c047-4d1d-8d63-e3e7005ccef1.png">

https://github.com/easy2use/gearbox-api-docs/blob/master/endpoints/services.md#request